src/QuadForm/Torsion.jl

```diff
@@ -409,7 +409,7 @@ mutable struct TorQuadModMor <: Map{TorQuadMod, TorQuadMod, HeckeMap, TorQuadMod
   header::MapHeader{TorQuadMod, TorQuadMod}
   map_ab::GrpAbFinGenMap
 
-  function TorQuadModMor(T::TorQuadMod, S::TorQuadMod, m::GrpAbFinGenMap)
+  function e(T::TorQuadMod, S::TorQuadMod, m::GrpAbFinGenMap)
```

Comment:
This change does look suspicious :)

Reply:
Oh... I think I accidentally did something stupid and didn't realise it. Will change it back and try again. Thanks :)